client/src/modules/cohort/components/Cohorts.jsx

```diff
@@ -122,9 +129,11 @@ class Cohorts extends Component {
               onClick={this.handleArchivedCohortsVisibility}
               type="button"
             >
-              {` ${
-                areArchivedCohortsVisible ? 'hide' : 'show'
-              } archived cohorts`}
+              {areArchivedCohortsVisible ? (
+                <FormattedMessage id="cohort.cohorts.hide-archived" />
```

Comment:
why `cohort.cohorts` prefix? if this is string from cohorts then just `cohort` should be enough

Reply:
because I took the approach of "module.component-name.string-id"
and here we are in **cohort** module, cohorts **componenets**, and string is **hide-archived**